src/services/__mocks__/Api.ts

```diff
@@ -124,3 +125,11 @@ export const getNamespaceMetrics = (namespace: String, params: any) => {
     }
   });
 };
+
+export const GetErrorMsg = (msg: string, error: AxiosError) => {
```

Comment:
Is that connected to the Pull Request? Perhaps commited by error? Otherwise, what is it for?

Reply:
Because the ServiceInfo in some cases when you test the catch fail because this methods is not in the mock.